src/testing/native-debugging.md

```diff
@@ -0,0 +1,459 @@
+| **Orange**                    | Problems            | List of issues the Flutter linter found in the current Flutter file               |
+|                               | Output              | Response that the Flutter app                                                     |
+|                               | Debug Console       | Logs or error messages that the Flutter app generates while debugging             |
+|                               | Terminal            | System shell prompt contained in VS Code                                          |
+{:.table.table-striped}
+</div>
+
+To change where the panel (in **orange**) appears in VS Code,
+go to **View** > **Appearance** > **Panel Position**.
+
+#### VS Code Flutter debugging toolbar
+
+The toolbar allows you to debug as any debugger.
+You can step in, out, and over Dart statements, hot reload or resume the app.
+
+![Flutter debugger toolbar in VS Code](/assets/images/docs/testing/debugging/vscode-ui/screens/debug-toolbar.png)
+
+<div class="table-wrapper" markdown="1">
+| Icon                                                      | Action                | Keyboard Shortcut                                     |
+|-----------------------------------------------------------|-----------------------|-------------------------------------------------------|
+| {% include_relative vscode-flutter-bar/_play.md %}        | Start or Resume       | <kbd>F5</kbd>                                         |
+| {% include_relative vscode-flutter-bar/_pause.md %}       | Pause                 | <kbd>F5</kbd>                                         |
+| {% include_relative vscode-flutter-bar/_step-over.md %}   | Step Over             | <kbd>F10</kbd>                                        |
+| {% include_relative vscode-flutter-bar/_step-into.md %}   | Step Into             | <kbd>F11</kbd>                                        |
+| {% include_relative vscode-flutter-bar/_step-out.md %}    | Step Out              | <kbd>Shift</kbd> + <kbd>F11</kbd>                     |
+| {% include_relative vscode-flutter-bar/_hot-reload.md %}  | Hot Reload            | <kbd>Control</kbd> + <kbd>F5</kbd>                    |
+| {% include_relative vscode-flutter-bar/_hot-restart.md %} | Hot Restart           | <kbd>Shift</kbd> + <kbd>Special</kbd> + <kbd>F5</kbd> |
+| {% include_relative vscode-flutter-bar/_stop.md %}        | Stop                  | <kbd>F5</kbd>                                         |
+| {% include_relative vscode-flutter-bar/_inspector.md %}   | Open Widget Inspector |                                                       |
+{:.table.table-striped}
+</div>
+
+[Test drive]: {{site.url}}/get-started/test-drive
+
+## Update test Flutter app
+
+For the remainder of this guide, you need to update the
+test Flutter app. This update adds native code to debug.
+
+1. Open the `lib/main.dart` file using your preferred IDE.
```

Comment:
```suggestion
1. Open the `lib/main.dart` file using your preferred editor.
```

Reply:
Would not IDE be correct? VS code is an IDE.